defaults/main.yml

```diff
@@ -486,6 +486,11 @@ apt__apt_preferences__dependent_list:
     reason:  'Issues in wheezy package - http://debian.distrosfaqs.org/debian-user/wheezy-irqbalance/'
     by_role: 'debops.apt'
 
+  - package: 'irqbalance'
+    backports: [ 'jessie' ]
```

Comment:
@drybjed I decided against this as the reasons for the two releases differ. I would prefer keeping the real reason for each release. What do you think?